cl/opinion_page/templates/view_opinion.html

```diff
@@ -148,6 +148,29 @@ <h3><span>Visualizations</span></h3>
             </div>
         {% endif %}
 
+        {# Recommendations #}
```

Comment:
This should definitely go above visualizations. I also wonder if the cases citing and cases cited sections should be merged with this section into a common template? You could use a `with` statement sent to a template include to make these all use the same template, I think? https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#include